### ISSUE TITLE
Porting to VC6

### ISSUE THREAD

**Issue by peyton-howe** [Seneral/VC4CV#3]
Hey Seneral, I have finally gotten my hands on a Pi 4 and am looking to port my previous work on the Pi 3 over. With the new GPU on the Pi 4, I know there is probably a good bit to convert and the first step is to change the windowing from dispmanx to SDL2 since dispmanx is no longer supported. I think I have it implemented correctly, but I’m running into a new compiling error that I’m not sure how to fix. I’m guessing it’s due to the makefile settings, but I’m not sure how to modo if you them.

Here is the compiling issue:
![188F305F-F6AA-45CD-BA3F-D613A67C7386](https://user-images.githubusercontent.com/80078168/168900112-3acee60e-2e5c-4fb8-9cc5-d4de9399e8f6.jpeg)

**Comment by Seneral** [comment]
Well you already know more than I do, thought GL code might just port over, whereas QPU is basically a rewrite. So sorry, can't help, never dealt with a Pi 4 before

But since that is the linker, you need to add the SDL libraries into the cmake, somehow, just look it up

**Comment by peyton-howe** [comment]
Fair enough. I have gotten past the make issues and am now running into issues regarding the eglCreateImageKHR() function. 

Did you have a lot of issues with this function when you were initially creating this program? If so, was there an easy way to debug it or any good resources besides the khronos documentation?
